doc/source/reference/executables/grndb.rst

```diff
@@ -118,6 +118,18 @@ Here is an example that specifies ``--log-path`` option::
 
   % grndb check --log-path /var/log/groonga/grndb.log /var/lib/groonga/db/db
 
+``--log-flags``
+"""""""""""""""
+
+.. versionadded:: 9.0.2
+
+It specifies a flag of ``grndb`` log.
```

Comment:
In my opinion, we should write about we can output what.

Reply:
Thanks!
I've fixed.